build.sh

```diff
@@ -7,8 +7,10 @@ set -e -u -o pipefail
 # ref: https://cmake.org/cmake/help/latest/variable/CMAKE_CUDA_ARCHITECTURES.html
 declare -r CMAKE_CUDA_ARCHITECTURES="${CUDAARCHS:-native}"
 
-legate_root=`python -c 'import legate.install_info as i; from pathlib import Path; print(Path(i.libpath).parent.resolve())'`
+legate_root=$(
+    python -c 'import legate.install_info as i; from pathlib import Path; print(Path(i.libpath).parent.resolve())'
+)
 echo "Using Legate at $legate_root"
-cmake -S . -B build -D legate_core_ROOT=$legate_root -DCMAKE_BUILD_TYPE=Release -DCMAKE_CUDA_ARCHITECTURES=${CMAKE_CUDA_ARCHITECTURES}
+cmake -S . -B build -Dlegate_core_ROOT="${legate_root}" -DCMAKE_BUILD_TYPE=Release -DCMAKE_CUDA_ARCHITECTURES="${CMAKE_CUDA_ARCHITECTURES}"
```

Comment:
the changes in this file are all to satisfy `shellcheck`. It encourages the use of `$()` over backticks, and quoting expansions in case the expanded variables include spaces.